docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss

```diff
@@ -57,20 +59,125 @@
   }
 
   &--light {
+    .hb-color-band--title {
+      @include hb-global-color('color', 'black');
+    }
+
     .hb-color-band--text {
       @include hb-global-color('color', 'black');
     }
+
+    .hb-color-band--link .hs-more-link a,
+    .hb-color-band--link.hs-more-link a,
+    .hb-color-band--link .hs-more-link button,
+    .hb-color-band--link.hs-more-link button {
+      @include hb-global-color('color', 'black');
+
+      &:hover,
+      &:focus {
+        @include hb-global-color('color', 'black');
+
+        &::after {
+          @include hb-icon-link-arrow('black');
+
+          background-size: 85%;
+
+          *[class^="hc-pairing-"] &,
+          *[class^="ht-pairing-"] & {
+            @include hb-icon-link-arrow('black');
+
+            background-size: 85%;
+          }
+        }
+      }
+    }
+
+    .hb-color-band--link .hs-more-link a::after,
+    .hb-color-band--link.hs-more-link a::after,
+    .hb-color-band--link .hs-more-link button::after,
+    .hb-color-band--link.hs-more-link button::after {
+      @include hb-icon-link-arrow('black');
+
+      background-size: 85%;
+
+      *[class^="hc-pairing-"] &,
+      *[class^="ht-pairing-"] & {
+        @include hb-icon-link-arrow('black');
+
+        background-size: 85%;
+      }
+    }
+  }
+
+  &--title {
+    @include hb-global-color('color', 'white');
+
+    margin: 0 0 hb-calculate-rems(12px);
   }
 
   &--text {
     @include hb-global-color('color', 'white');
 
+    margin: 0;
+    padding: 0 hb-calculate-rems(12px) hb-calculate-rems(12px);
+    font-size: hb-calculate-rems(22px);
+    font-weight: hb-theme-font-weight(semibold);
+    line-height: 127%;
+
     @include hb-themes(('colorful', 'airy')) {
-      padding: hb-calculate-rems(12px);
+      padding: 0 hb-calculate-rems(12px) hb-calculate-rems(12px);
     }
 
     @include hb-traditional {
-      padding: hb-calculate-rems(20px);
+      padding: 0 hb-calculate-rems(20px) hb-calculate-rems(20px);
```

Comment:
Why 20px? It should be 12px according to the design spec.
<img width="1018" alt="Screenshot 2024-01-30 at 18 20 16" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/4eccfb98-dcd4-4ed9-96ba-d157da4296e5">

Reply:
I think I had noticed that there was a traditional variation that was larger padding ... but, yes, I'll remove that.